### PR TITLE
Data tab data fetching and basic UI implementation

### PR DESCRIPTION
**Pull request by vaidotasp**
* Part of #782

- Imports analysis svg icon from PROD site
- mapController fires of popup watch utils that hook into ESRI popup functionality to listen to features user clicked on
- Custom async fetching logic for server side features
- Redux changes to track "activeFeatures", meaning features that user clicked on!
- Extremely basic implementation of data tab showing off different attributes for active features
- Change tsconfig to allow strictPropertyInitialization. This way we, for the most part, do not need initialize undefined values in mapController. See _mapview example.

### CODE REVIEW

File: src/js/components/leftPanel/DataTabView.tsx [hunk 1]
```diff
@@ -1,19 +1,85 @@
...
+    return activeFeatures.map((layerFeatureGroup, i) => (
+      <div key={i}>
+        <FeatureGroupElement layerFeatureGroup={layerFeatureGroup} />
+      </div>
+    ));
+  };
+
+  const DataTabViewContent = () => {
+    if (!tabViewIsVisible) {
+      return <></>;
```
Comment (SwampGuzzler) [inline src/js/components/leftPanel/DataTabView.tsx]: Return `null`

**Review by SwampGuzzler — CHANGES_REQUESTED** [review]
Let's change our "watch(mapview.popup, 'promises'," etc to use a `once` the exact same way, then we don't have to remove our `popupWatcher` --> https://developers.arcgis.com/javascript/latest/api-reference/esri-core-watchUtils.html#once

File: src/js/helpers/DataPanel.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      returnGeometry: false
+    };
+    try {
+      const sublayerResult = await esriQuery(url, qParams);
+      //if no features are found, skip it
+      if (sublayerResult.features.length !== 0) {
+        const features = sublayerResult.features.map(f => {
+          return {
+            attributes: f.attributes,
+            geometry: f.geometry
```
Comment (SwampGuzzler) [inline src/js/helpers/DataPanel.ts]: Can we hard-code `null` here instead of f.geometry ? Or add a comment

File: src/js/helpers/DataPanel.ts [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        promises.map((p: Promise<any>) => p.catch((error: Error) => null))
+      ).then(async clientFeatures => {
+        let layerFeatureResults: LayerFeatureResult[] = [];
+        const cleanClientFeautures = clientFeatures
+          .filter(f => f !== null) //catch failed promises
+          .filter(f => f.length !== 0) //catch no features returned
+          .map((featureObject: Graphic[]) => {
+            const newFeatureObject: LayerFeatureResult = {
+              layerID: featureObject[0].layer.id,
+              layerTitle: featureObject[0].layer.title,
```
Comment (SwampGuzzler) [inline src/js/helpers/DataPanel.ts:111]: Is `this` the place to add a PopupTemplate for each layer? Should that be in `allAvailableLayers` in State?

**Comment by vaidotasp** [comment]
@SwampGuzzler implemented requests for changes, 
0701ae7 and cb26ce8

**Review by SwampGuzzler — APPROVED** [review]
Swell!

**Pull request merged**